### DISCUSSION TITLE
Is there a project with open source translations that you can reuse?

### DISCUSSION THREAD

**Discussion by romanr** [WeblateOrg/weblate#11349] [General]
Every project has a lot of same translations to same languages: number names, colors, "Ok", "Cancel" , "sign up", same errors and informational messages.
Majority, if not 90% in some cases, of projects use same labels and same messages. 

**Is there a repository/project that hosts common language translations, so we can translate once and reuse across community rather than each one of us, separately redoing same translations with every project or module?** 

I know the "translation memory" automatic option in Weblate, and it's been great, providing good quality translations. But you have to go over entire translation, every key, and click and click.

**Comment by emmapeel2** [comment]
You can activate the 'Automatic translations' add on and make it add translations with a certain threshold.
